cpp/src/parquet/column_writer.h

```diff
@@ -136,8 +138,15 @@ class PARQUET_EXPORT ColumnWriter {
 
   /// \brief The total number of bytes written as serialized data and
   /// dictionary pages to the ColumnChunk so far
+  /// These bytes are uncompressed bytes.
   virtual int64_t total_bytes_written() const = 0;
 
+  /// \brief The total number of bytes written as serialized data and
+  /// dictionary pages to the ColumnChunk so far.
+  /// If the column is uncompressed, the value would be equal to
```

Comment:
Is this true? Will total_compressed_bytes_written() be greater than total_bytes_written() because of page headers and column chunk metadata?

Reply:
Column Chunk Metadata will not be added to count. And both the compressed size and uncompressed would be written to column chunk.

page header will not be compressed.

Reply:
Do both of them record bytes from dictionary pages written?